packages/fast-components-react-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import MSFTPivot, {
+    PivotHandledProps as MSFTPivotHandledProps,
+    PivotManagedClasses,
+    PivotProps as MSFTPivotProps,
+    PivotUnhandledProps,
+} from "./pivot";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import manageJss, { ManagedJSSProps } from "@microsoft/fast-jss-manager-react";
+import { DesignSystem, PivotStyles } from "@microsoft/fast-components-styles-msft";
+import { Subtract } from "utility-types";
+
+/*tslint:disable-next-line:typedef */
+const Pivot = manageJss(PivotStyles)(MSFTPivot);
+type Pivot = InstanceType<typeof Pivot>;
+
+interface PivotHandledProps
+    extends Subtract<MSFTPivotHandledProps, PivotManagedClasses> {}
+type PivotProps = ManagedJSSProps<MSFTPivotProps, PivotClassNameContract, DesignSystem>;
+
+export { Pivot, PivotProps, PivotHandledProps, PivotManagedClasses, PivotUnhandledProps };

```

Comment:
So this is where we would want to export the `PivotClassNameContract`, we do not want to export `PivotManagedClasses`.

What the user needs from this export is:
- The component (Pivot)
- The full prop contract (PivotProps)
- The class name contract (PivotClassNameContract)
- The unhandledProps (PivotUnhandledProps)
- The handledProps (PivotHandledProps)
- any other type definition inside of the props that would be useful, such as enums, this component does not appear to have any